src/components/Menu/MenuItem.tsx

```diff
@@ -97,4 +132,17 @@ export const MenuItem = <C extends MenuItemElement = "button">({
       {children}
     </Component>
   );
+
+  if (interactive) {
+    switch (context?.component) {
+      case "Radix dropdown menu":
```

Comment:
What happens if `interactive` is `true` and the component used is `vaul` ?
I don't quite follow this bit?